pkg/network/render_test.go

```diff
@@ -33,3 +35,48 @@ func TestIsChangeSafe(t *testing.T) {
 	err = IsChangeSafe(prev, next)
 	g.Expect(err).To(MatchError(ContainSubstring("cannot change default network type")))
 }
+
+func TestRenderUnknownNetwork(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	config := netv1.NetworkConfig{
+		Spec: netv1.NetworkConfigSpec{
+			ServiceNetwork: "172.30.0.0/16",
+			ClusterNetworks: []netv1.ClusterNetwork{
+				{
+					CIDR:             "10.128.0.0/15",
+					HostSubnetLength: 9,
+				},
+				{
+					CIDR:             "10.0.0.0/14",
+					HostSubnetLength: 8,
+				},
+			},
+			DefaultNetwork: netv1.DefaultNetworkDefinition{
+				Type: "MyAwesomeThirtPartyPlugin",
```

Comment:
Yes, it doesn't really matter, but I know the typo there ("`Thirt`") is eventually going to annoy me...

Reply:
We can't have that.